Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPDistanceTraits.h

```diff
@@ -0,0 +1,106 @@
+/// \ingroup PkgPolygonMeshProcessingConcepts
+/// \cgalConcept
+///
+/// The concept `PMPDistanceTraits` is a refinement of the
+/// concepts `AABBGeomTraits` and `SpatialSortingTraits_3`. In addition to the types required by
+/// those concepts,  it also requires types and functors necessary to
+/// define the functions `approximate_max_distance_to_point_set()`, `sample_triangle_mesh()`,
+/// `approximate_Hausdorff_distance()` and `max_distance_to_triangle_mesh()`
+///
+/// \cgalRefines `AABBGeomTraits` `SpatialSortingTraits_3`
```

Comment:
IIRC correctly, you can put 
```
\cgalRefines `AABBGeomTraits`
\cgalRefines `SpatialSortingTraits_3`
```
Please give it a try. If it does render nicely put _and_ between concepts refined.